### PR TITLE
Players now start facing up ⬆️ + Displayed player coordinates 🔢 

### LINKED ISSUE #298: Initial angle should be facing upwards
Currently red has a advantage over the green because he can just zoom whilst green has to turn before. Make the dragons face upwards when the game starts.

### PR DESCRIPTION
**Pull request by TheoPannetier**
- Fixed #298, players now start with an angle of 270 degrees (facing up, 0 = facing right)
- Adapted tests that check movement and wall collisions work as expected
- Changed wall names to the more intuitive cardinal directions 🧭. 
Note that I renamed "upper" and "lower" walls to south and north resp. "Upper" wall (y = max_y) was actually the bottom one on-screen, and "lower" wall was the top one (y = 0)

### CODE REVIEW

**Comment by TheoPannetier** [comment]
There's more stuff now, the players coordinates are displayed in the top-right quarter of the window 👍

File: game_view.cpp [hunk 1]
```diff
@@ -259,6 +282,18 @@ void game_view::show() noexcept
...
     }
 
+    // Set fourth view for players coordinates
+    #ifndef NDEBUG  // coordinates should not be visible in release
+    sf::View player_coords_view(
+                sf::Vector2f(m_window.getSize().x / 4.5, m_window.getSize().y / 4.5),
+                sf::Vector2f(m_window.getSize().x / 2, m_window.getSize().y / 2)
+                );
+    player_coords_view.setViewport(sf::FloatRect(0.5f, 0.0f, 0.5f, 0.5f));
+    m_window.setView(player_coords_view);
```
Comment (swom) [inline game_view.cpp:292]: You should make this a function `game_view::set_player_coords_view()` and maybe call it inside `draw_player_coords()`

**Review by swom — CHANGES_REQUESTED** [review]
Overall very nice work, and thanks for fixing my typos.
It would if you could address my comment.

**Review by ollyturner — COMMENTED** [review]
Great Job Well Done!

**Comment by ollyturner** [comment]
Sorry I forgot to click send

**Pull request merged**